### PR TITLE
Activity feed broadcast nodejs

### PR DESCRIPTION
**Pull request by NikitaJain-MSFT**

### CODE REVIEW

File: samples/graph-activity-feed-broadcast/nodejs/server.js [hunk 1]
```diff
@@ -0,0 +1,256 @@
...
+              "accept": "application/json",
+              "contentType": 'application/json',
+              "authorization": "bearer " + applicationToken
+            }
+          })
+          .then(res => {
+            console.log(`statusCode: ${res.status}`)
+            if(res.status == 429)
+            {
+              polly()
```
Comment (AbhijeetThakur-msft) [inline samples/graph-activity-feed-broadcast/nodejs/server.js:215]: Shall we move this code to common method?

**Review by AbhijeetThakur-msft — CHANGES_REQUESTED** [review]

File: samples/graph-activity-feed-broadcast/nodejs/server.js (hunk 1 above)
Comment (NikitaJain-MSFT) [reply samples/graph-activity-feed-broadcast/nodejs/server.js]: moved to a common method

File: samples/graph-activity-feed-broadcast/nodejs/server.js [hunk 2]
```diff
@@ -0,0 +1,242 @@
...
+    title: req.body.title,
+    description: req.body.description,
+    createdBy: req.body.userName,
+    userId: req.body.userId
+  };
+
+  localdata.push(taskDetails);
+  var appId;
+
+  axios.get("https://graph.microsoft.com/v1.0/users/" + req.body.userId + "/teamwork/installedApps/?$expand=teamsAppDefinition", {
```
Comment (Wajeed-msft) [inline samples/graph-activity-feed-broadcast/nodejs/server.js:106]: Could you please use the Graph SDK wherever possible?

File: samples/graph-activity-feed-broadcast/nodejs/server.js [hunk 3]
```diff
@@ -0,0 +1,242 @@
...
+
+  axios.get("https://graph.microsoft.com/v1.0/users", {
+    headers: {
+      "accept": "application/json",
+      "contentType": 'application/json',
+      "authorization": "bearer " + delegatedToken
+    }
+  })
+  .then(userList => {
+    for (let i = 0; i < userList.data.value.length; i++) {
```
Comment (Wajeed-msft) [inline samples/graph-activity-feed-broadcast/nodejs/server.js:128]: Check how this can be done parallelly?

**Review by Wajeed-msft — CHANGES_REQUESTED** [review]

File: samples/graph-activity-feed-broadcast/nodejs/server.js [hunk 4]
```diff
@@ -0,0 +1,242 @@
...
+
+  // Creating MSAL client
+  const msalClient = new msal.ConfidentialClientApplication({
+    auth: {
+      clientId: process.env.ClientId,
+      clientSecret: process.env.ClientSecret
+    }
+  });
+
+  var oboPromise = new Promise((resolve, reject) => {
```
[the remark this answers is not in the document]
Comment (NikitaJain-MSFT) [reply samples/graph-activity-feed-broadcast/nodejs/server.js:74]: we are not passing token back to client all graph calls are made on server side

File: samples/graph-activity-feed-broadcast/nodejs/server.js (hunk 2 above)
Comment (NikitaJain-MSFT) [reply samples/graph-activity-feed-broadcast/nodejs/server.js]: updated code with graph sdk method

File: samples/graph-activity-feed-broadcast/nodejs/server.js (hunk 3 above)
Comment (NikitaJain-MSFT) [reply samples/graph-activity-feed-broadcast/nodejs/server.js]: added parallel-each package

**Review by AbhijeetThakur-msft — APPROVED** [review]

**Review by v-chetsh — APPROVED** [review]
Changes looks fine, approving it.

**Review by Wajeed-msft — APPROVED** [review]

**Pull request merged**